### DISCUSSION TITLE
Equipo 3

### DISCUSSION THREAD

**Discussion by jennyperez28** [PatronesHermosos/design-thinking-1ra-generacion#21] [Q&A]
Hola Kasturi y equipo de Patrones Hermosos.
Las integrantes del equipo 3 hemos estado hablando y avanzando en la Etapa 0. 
Podrian revisarla porfa. Consideramos que a esta lista para pasar a la Etapa siguiente.

Gracias

**Comment by KasturiSShah** [comment]
Hola @jennyperez28 
Gracias por avisar. He puesto mi retroalimentación en tu hilo grupal https://github.com/PatronesHermosos/design-thinking-1ra-generacion/discussions/8
Cualquier cosa, me etiquetas en el hilo grupal y te estaré contestando 😄 
Kasturi
